oak_attestation_verification/src/rekor.rs

```diff
@@ -136,22 +144,13 @@ impl TryFrom<&LogEntry> for RekorSignatureBundle {
     type Error = anyhow::Error;
 
     fn try_from(log_entry: &LogEntry) -> anyhow::Result<Self> {
-        // Create a copy of the LogEntry, but skip the verification.
-        let entry_subset = LogEntry {
-            body: log_entry.body.clone(),
-            integrated_time: log_entry.integrated_time,
-            log_id: log_entry.log_id.clone(),
-            log_index: log_entry.log_index,
-            verification: None,
-        };
-
-        // Canonicalized JSON document that is signed. Canonicalization should follow the RFC 8785
-        // rules.
-        let mut serializer = Serializer::with_formatter(Vec::new(), CanonicalFormatter::new());
-        entry_subset
-            .serialize(&mut serializer)
-            .context("Failed to serialize Rekor signed payload to JSON")?;
-        let signed_json_bytes: Vec<u8> = serializer.into_inner();
+        // Canonicalized JSON document that is signed; note that verification is omitted.
+        // Canonicalization should follow the RFC 8785 rules. We hardcode the canonical
+        // serialization because serialization with serde_json requires std.
+        let canonicalized = format!(
+            "{{\"body\":\"{}\",\"integratedTime\":{},\"logID\":\"{}\",\"logIndex\":{}}}",
```

Comment:
I may be wrong, but I think you need a closing " before the second comma. If I'm right, it would be nice to see a failing test for this (or file an issue to add such test and assign to me please)

Reply:
Oh, ignore that, integratedTime is an integer.